packages/fxa-graphql-api/src/lib/db/models/auth/emails.ts

```diff
@@ -0,0 +1,21 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { AuthBaseModel } from './auth-base';
+
+export class Emails extends AuthBaseModel {
+  public static tableName = 'emails';
+
+  protected $uuidFields = ['uid', 'emailCode'];
```

Comment:
Are the `$uuidFields` unique columns in table? The `emailCode` is not guaranteed to be, but `normalizedEmail` is.

Reply:
This ORM is a bit lighter-weight than others, in that you're only reflecting the types you use here, not actually defining any DDL for the underlying schema. `$uuidFields` is just a class variable used by the BaseModel to know which data to turn to/from buffers for binary before saving and after loading.